### ISSUE TITLE
Allow launcher use while offline

### ISSUE THREAD

**Issue by bigfoot547** [olauncher/olauncher#31]
Launcher should allow use while offline (as long as there is a UUID and name to use). Obviously the session token will be invalid (find a way to make the launcher okay with this).

**Comment by FoxbitDreamtail** [comment]
You know, this is still an issue. On real hardware Windows XP, I cannot log in in spite of being logged in before online.

**Comment by bigfoot547** [comment]
yeah I think it's a regression in 183b744ef9ccf63d0db1d9c8f16676d723f05ed5

**Comment by FoxbitDreamtail** [comment]
> yeah I think it's a regression in [183b744](https://github.com/olauncher/olauncher/commit/183b744ef9ccf63d0db1d9c8f16676d723f05ed5)

When do you think a patch would be available in release? I'm only asking because I don't want to fight Multi-MC on Windows XP, And I want the nostalgic look to remain.

**Comment by bigfoot547** [comment]
I'm not sure, however I promised that the next bug I would work on is the resource-mapped bug related to pre-1.6 game assets so this will have to be fixed after that.

**Comment by LOuroboros** [comment]
I realized today that even though I had already logged into my Microsoft/MC account, I couldn't use the launcher to play MC at all. It just gets stuck on the initial loading screen, with the Play button grayed out and such, so I wanted to ask if this will be addressed in the foreseeable future.

**Comment by Consumedgrub2** [comment]
Bump. I just lost internet myself but I thought no big deal, it's the old school launcher, it should be able to play offline like the good old days unlike the crappy bloated slow new always online, must always be logged into Microsoft account and logs you out after awhile launcher, but nope, same issue as comments above.

**Comment by LOuroboros** [comment]
From the repo's README.

> Project Status
> This project is on indefinite hiatus. Check the forks tab to see if anyone is maintaining a fork of this project. I have relicensed olauncher as CC0 with the consent of the contributors, so you are free to distribute any forks with whatever license you see fit.

In short, we're cooked.

The few forks that are in active development that can be seen at [Insights](https://github.com/olauncher/olauncher/pulse) -> [Forks](https://github.com/olauncher/olauncher/forks) have their Issues tab disabled, and one of them specified that there's no plans to introduce any features or make changes beyond keeping the launcher updated compatibility wise.
